ui/src/routes/about_us.tsx

```diff
@@ -44,7 +44,7 @@ function AboutUs() {
   return (
     <Layout className="layout">
       <Helmet>
-        <meta name="description" content={HOME_PAGE_METATAG_CONTENT} />
```

Comment:
we shouldn't remove the description.

Reply:
I change the tag `meta` by `title` in the other pr

Reply:
hmm what do you mean? the other PR is for the picture. We shouldn't remove the `description`

Reply:
Oh sorry I thought I was on the other pr, forgot what I said